README.md

```diff
@@ -157,3 +166,31 @@ models:
             - ref('raw_customers')
     columns: ...
 ```
+
+### Support multiple test cases
+
+It is also possible to support the same model unit test over multiple test cases. Keeping the yaml config concise, but still easily identifying the specific test cases that are failing.
+
+* `test_case_list`: new optional field, expects an array with the test cases to be applied on `input_mapping` and `expected_output`
+* `@`: The special character to be used as template for string substitution, iterating through the values of `test_case_list`
+* `\`: the special character added to the beginning of the value. This is required to stop Jinja automatically rendering the value (i.e: `ref()`, `source()` etc.)
+
+```yaml
+models:
+  - name: stg_customers
+    tests:
+      - dbt_datamocktool.unit_test:
+          tags: 
+            - example_tag_unit_test 
+          test_case_list: [1,2,3,5,8,13,21]
+          input_mapping:
+            source('jaffle_shop', 'raw_customers'): \ref('dmt__raw_customers_@')
+          expected_output: \ref('dmt__expected_stg_customers_@')
+```
+
+This will result in a **single** test being executed. But under the hood, all test cases listed are checked. Any error count is passed as the test output.
+
+In case of any failures, the **single** test will return a count of errors > 0, making it a "dbt-test failure". The full log will contain a list of all failing tests.
+
+### Visibility on test failure
+TODO: if storing result as a view, it is possible to inspect the mocked model code, making it easier to debug issues with failing tests.
```

Comment:
To be further fleshed out if okay to move forward with storing mocked model output